test-it/GetObjPathArrayIndex.via

```diff
@@ -0,0 +1,61 @@
+// Autogenerated Vireo assembly file
+// Generated on Tue Oct 08 15:12:36 2013
+define(Point c(e(.Int32 x)e(.Int32 y)))
+//---------------------------------------------------
+//VI Definition: stringManipulation.vi
+define(videf1  dv(.VirtualInstrument (
+    c(
+        e(dv(.Int32 0) c1)   // index
+        e(dv(.String 'brown fox ') c2)   // new element/subarray
+        e(dv(.String 'The quick ') c3)   // new element/subarray
+        e(dv(.String 'Blue Fish.') c4)   // new element/subarray
+        e(dv(.String 'Red Fish, ') c5)   // new element/subarray
+        e(dv(.String 'the lazy dog.') c6)   // element
+        e(dv(.String 'jumps over ') c7)   // element
+        e(dv(.String 'One Fish, ') c8)   // element
+        e(dv(.Int32 2) c9)   // index
+        e(dv(.String 'Two Fish. ') c10)   // element
+        e(a(.String *) w11)
+        e(a(.String *) w13)
+        e(.String w15)
+        e(a(.String *) w16)
+        e(dv(a(.Int32 *) (1 2 3 4 5) ) arrint)
+        e(dv(a(Point *) ((1 2) (10 20) (100 200))) arrclust)
+        e(dv(a(.Int32 * *) ((11 12 13 14 15) (21 22 23 24 25)) ) arr2d)
+        e(dv(.Int32 0) i1)
+        e(dv(Point) clust)
+        e(.String w17)
+        e(.Type ty)
+    )
+    clump(1 
+        ArrayConcatenate( w11 c3 c2 )
+        ArrayConcatenate( w13 c8 c10 c7 c6 )
+        Println(w15 )
```

Comment:
I disagree.  Hard-coded line numbers make it hard to add new cases to the tests in the middle near other similar cases.  I think this sort of issue should be fixed in the test-runner infrastructure; it can easily provide differing line numbers or other context.  The test.js runner already does, even with colorized diffs.